gulpfile.js

```diff
@@ -60,13 +62,11 @@ gulp.task('watch', function (done) {
     sequence('build-static', ['watchify', 'watchify-publish', 'watch-less'], 'cache', done);
 });
 
-// Test
 gulp.task('jshint', jshint);
 gulp.task('jscs', jscs);
 gulp.task('lint', ['jshint', 'jscs']);
 gulp.task('unit', ['build'], unit);
 gulp.task('test', ['lint', 'unit']);
-gulp.task('svgmin', svgmin);
```

Comment:
that is an excellent question, I don't remember removing that so chucking it up to "accidental cmd-x" and will make sure that goes back in.